### PR TITLE
Install Camel LS through CocInstall. 

### PR DESCRIPTION
**Pull request by Navyashree0923**
I have created a  package and updated to npm registry using `npm publish` https://www.npmjs.com/package/coc-camel. Now we can able to install Camel LS through `CocInstall`.

### CODE REVIEW

File: .github/workflows/coc-camel-dev.yaml [hunk 1]
```diff
@@ -19,7 +19,6 @@ jobs:
     - uses: actions/setup-node@v3
       with:
             node-version: 16
-            cache: 'yarn'
```
Comment (apupier) [inline .github/workflows/coc-camel-dev.yaml:22]: why removing the cache?

File: package.json [hunk 2]
```diff
@@ -1,7 +1,11 @@
 {
   "name": "coc-camel",
-  "version": "0.0.0-development",
```
Comment (apupier) [inline package.json:3]: version 0.0.0 is already released on the npm registry. I think it is better to use an higher version, like 0.1.0

**Review by apupier — CHANGES_REQUESTED** [review]

File: .github/workflows/coc-camel-dev.yaml (hunk 1 above)
Comment (Navyashree0923) [reply .github/workflows/coc-camel-dev.yaml:22]: > why removing the cache?

I have removed yarn.lock in this PR. When the workflow is running is without yarn.lock it was failing with below errors . To fix the errors i have removed the cache line
`Error: Dependencies lock file is not found in /home/runner/work/coc-camel/coc-camel. Supported file patterns: yarn.lock`

File: package.json (hunk 2 above)
Comment (Navyashree0923) [reply package.json:3]: Thank you. I have already included this PR changes in v0.0.0

File: .github/workflows/coc-camel-dev.yaml (hunk 1 above)
Comment (apupier) [reply .github/workflows/coc-camel-dev.yaml:22]: Why removing yarn.lock then?

File: package.json (hunk 2 above)
Comment (apupier) [reply package.json:3]: Should try to never release something which as not pushed to the repository.
I think that now we can let it go with 0.0.0 version and need to increment the version to 0.1.0 anyway

File: .github/workflows/coc-camel-dev.yaml (hunk 1 above)
Comment (Navyashree0923) [reply .github/workflows/coc-camel-dev.yaml:22]: I have included yarn.lock file in the recent PR that was merged and when I do the PlugInstall with having yarn.lock file is producing the following errors. 
 ```                                                                            |
x Post-update hook for coc-camel ... Exit status: 1                                                |" Shorthand notation; fetches https://github.com/neoclide/coc.nvim
- coc-camel: Receiving objects: 100% (18/18), 143.88 KiB | 975.00 KiB/s, done.                     |Plug 'neoclide/coc.nvim', {'branch': 'release'}
- coc.nvim: Already installed  
```   
Errors were not there before the lock file so decided to remove the lock file from the repo to fix the errors.

File: package.json (hunk 2 above)
Comment (Navyashree0923) [reply package.json:3]: Sorry my bad. When I was running the PlugInstall with having yarn.lock file in the repo was producing the errors so decided to fix it and I have created this npm package later. Starting now i'll increment the version after merging the PR.

File: .github/workflows/coc-camel-dev.yaml (hunk 1 above)
Comment (apupier) [reply .github/workflows/coc-camel-dev.yaml:22]: fine to remove the lock for this time. Be aware that providing a .lock is a good practice of Software Maintenance. it allows having reproducible build. Without it, if we want need to rebuild the version, or something based on this version later, different dependencies version might be picked up. Which then can cause troubles if not compatible.

File: package.json (hunk 2 above)
Comment (apupier) [reply package.json:3]: I still think that you should increase the version number. the version that was published is not matching something in the repository so we should forget it, inlclude the required modifications and upgrade the version and then redo a release and create a correspondign tag.

Comment (Navyashree0923) [reply package.json:3]: Thank you for explaining it. I believe I found where it was causing the errors. I see errors below when I run yarn install.
```
yarn install v1.22.19
[1/5] Validating package.json...
warning coc-camel@0.0.0-development: The engine "coc" appears to be invalid.
[2/5] Resolving packages...
error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

shell returned 1
```
Now I have updated the new yarn.lock file

File: .github/workflows/coc-camel-dev.yaml (hunk 1 above)
Comment (Navyashree0923) [reply .github/workflows/coc-camel-dev.yaml:22]: Thank you for explaining it. I believe I found where it was causing the errors. I see errors below when I run yarn install.
```
yarn install v1.22.19
[1/5] Validating package.json...
warning coc-camel@0.0.0-development: The engine "coc" appears to be invalid.
[2/5] Resolving packages...
error Your lockfile needs to be updated, but yarn was run with `--frozen-lockfile`.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

shell returned 1
```
Now I have updated the new yarn.lock file

File: package.json (hunk 2 above)
Comment (Navyashree0923) [reply package.json:3]: Thank you for the feedback. I have increased the version number and also updated the workflow file for providing autopublish and autotag feature. Please I'll work with you this afternoon to create a repository secret called `NPM_TOKEN` for enabling auto publish feature.

**Review by apupier — APPROVED** [review]

**Pull request merged**